### ISSUE TITLE
Regression Performance in Test sets

### ISSUE THREAD

**Issue by gwaygenomics** [broadinstitute/cell-health#40]
Based on #39 I am getting seemingly strange values for R2 and MSE in the test set for regression performance. This may be a bug, and, if so, will be important to track down!

**Comment by gwaygenomics** [comment]
OK! I ran some tests (PR incoming) and it does not look like these values are off. Also, from wikipedia [coefficient of determination](https://en.wikipedia.org/wiki/Coefficient_of_determination)

> There are cases where the computational definition of R2 can yield negative values, depending on the definition used. This can arise when the predictions that are being compared to the corresponding outcomes have not been derived from a model-fitting procedure using those data.

So, we expect R^2 to be zero if it predicts the expected value of y in data that was trained on, but the value can be negative when applying to data not used train model.

In otherwords, this happens in the test set when the regression sum of squares is greater than the total sum of squares. (test set will have a different median value) (R^2 = 1 - (SS_regression / SS_total))

**Comment by gwaygenomics** [comment]
Also, from https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html

> Best possible score is 1.0 and it can be negative (because the model can be arbitrarily worse). A constant model that always predicts the expected value of y, disregarding the input features, would get a R^2 score of 0.0.
